src/runtime/d3d12compute.cpp

```diff
@@ -1025,6 +996,54 @@ static void D3D12WaitForPix() {
 }
 #endif
 
+static ID3D12Device *D3D12CreateDeviceForAdapter(IDXGIAdapter1 *adapter) {
+    TRACELOG;
+
+    DXGI_ADAPTER_DESC1 desc = {};
+    if (FAILED(dxgiAdapter->GetDesc1(&desc))) {
+        d3d12_halt("Unable to retrieve information (DXGI_ADAPTER_DESC1) about the selectd adapter.");
+        return NULL;
+    }
+    char Description[128];
+    for (int i = 0; i < 128; ++i) {
+        Description[i] = desc.Description[i];
+    }
+    Description[127] = '\0';
+    TRACEPRINT("Device selected: " << Description << "\n");
+
+#if 0
```

Comment:
nit: please don't check in `#if 0` code. Use preprocessor symbols to indicate why it's disabled. e.g.

#define DO_DXGI_OUTPUT 0
#if DO_DXGI_OUTPUT ...